specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json

```diff
@@ -250,7 +251,7 @@
           "Pool list usage metrics": { "$ref": "./examples/PoolListUsageMetrics.json" }
         },
         "summary": "Lists the usage metrics, aggregated by pool across individual time intervals, for the specified account.",
-        "description": "If you do not specify a $filter clause including a poolId, the response includes all pools that existed in the account in the time range of the returned aggregation intervals.",
+        "description": "If you do not specify a $filter clause including a poolId, the response includes all pools that existed in the account in the time range of the returned aggregation intervals. If not specified, the startTime and endTime filters default to the start and end times of the last aggregation interval currently available; that is, only the last aggregation interval is returned.",
```

Comment:
>If not specified [](start = 200, length = 16)

If what is not specified?  Suggest something like: "If you do not specify a $filter clause including a startTime or endTime, these filters default to the start and end time of the last aggregation interval; that is..."